panel.go

```diff
@@ -398,6 +398,32 @@ type Target struct {
 	Dimensions map[string]string `json:"dimensions,omitempty"`
 	Period     string            `json:"period,omitempty"`
 	Region     string            `json:"region,omitempty"`
+
+	// For Stackdriver
+	AlignOptions       []StackdriverAlignOptions `json:"alignOptions,omitempty"`
+	AliasBy            string                    `json:"aliasBy,omitempty"`
+	MetricType         string                    `json:"metricType,omitempty"`
+	MetricKind         string                    `json:"metricKind,omitempty"`
+	Filters            []string                  `json:"filters,omitempty"`
+	AlignmentPeriod    string                    `json:"alignmentPeriod,omitempty"`
+	CrossSeriesReducer string                    `json:"crossSeriesReducer,omitempty"`
+	PerSeriesAligner   string                    `json:"perSeriesAligner,omitempty"`
+	ValueType          string                    `json:"valueType,omitempty"`
+	GroupBys           []string                  `json:"groupBys,omitempty"`
+}
+
+type StackdriverAlignOptions struct {
```

Comment:
Can we add a comment which says:
```
// StackdriverAlignOptions is ...
```
Maybe it would be even nice to explain in a 2nd sentence what's the relationship between this and `StackdriverAlignOption`.